pkg/users/usersGet.go

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package users
+
+import (
+	"context"
+
+	"github.com/galasa-dev/cli/pkg/embedded"
+	"github.com/galasa-dev/cli/pkg/galasaapi"
+	"github.com/galasa-dev/cli/pkg/spi"
+)
+
+func GetUsers(loginId string, apiClient *galasaapi.APIClient, console spi.Console) error {
+	var err error
+	var outputText []galasaapi.UserData
+
+	loginId, err = validateLoginIdFlag(loginId)
+	if err == nil {
+		outputText, err = getUserDataFromRestApi(loginId, apiClient)
+
+		if err == nil {
+			extractedUserObject := &outputText[0]
+			console.WriteString("id: " + extractedUserObject.GetLoginId())
+		}
+
+	}
+
+	return err
+}
+
+func getUserDataFromRestApi(
```

Comment:
As this method is called getUserData... I would expect it to return a single user data record, not an array.